sbndqm/dqmAnalysis/CRT/SBND/fcl/config_crt_dqm.fcl

```diff
@@ -1,47 +1,177 @@
+#include "crt_channel_list.fcl"
+
 BEGIN_PROLOG 
 
 streamCRT: {
+  
   module_type: BernCRTdqmSBND
+
+  Debug:                        true
+  CRTModuleLabel:               "daq"
+  CRTInstanceLabel:             "ContainerBERNCRTV2"
+  SPECTDCModuleLabel:           "daq"
+  SPECTDCInstanceLabels:        [ "TDCTIMESTAMP", "ContainerTDCTIMESTAMP" ]
+  DAQHeaderModuleLabel:         "daq"
+  DAQHeaderInstanceLabel:       "RawEventHeader"
+  SPECTDCT1Channel:             0
+  RawTSCorrection:              367000
+  BigHitADCThreshold:           600
+  BoardsRequiredForResetSpread: 100
+  RateNormalisation:            0.05
+
   metric_board_config: {
     groups: {
-    CRT_board:[88, 152, 156, 153, 159, 134, 135, 238, 155, 151, 150, 136, 157, 158, 182, 149, 73, 58, 60, 56, 169, 166, 24, 57, 33, 34, 61, 171, 59, 176, 170, 168, 172, 173, 160, 222, 220, 81, 85, 143, 162, 133, 132, 44, 147, 146, 131, 79, 206, 204, 200, 18, 174, 148, 163, 164, 165, 55, 120, 42, 138, 130, 197, 199, 202, 19, 198, 45, 203, 207, 82, 86, 90, 91, 99, 100, 93, 94, 95, 87, 97, 98, 77, 78, 83, 84, 104, 103, 102, 101, 111, 107, 106, 115, 105, 123, 50, 125, 126, 129, 114, 118, 48, 145, 119, 223, 36, 51, 71, 187, 16, 205, 35, 228, 75, 121, 40, 39, 128, 127, 178, 144, 180, 141, 179, 109, 116, 47, 49, 117, 185, 184, 183, 72, 216, 213, 212, 211, 210, 209 ]}
-    streams:[archiving]
+    CRT_board: [
+	88, 152, 156, 153, 159, 134, 135, 238, 155, 151, 150, 136, 157, 158, 182, 149, 73, 58, # North
+	60, 56, 169, 166, 24, 57, 33, 34, 61, 171, 59, 176, 170, 168, 172, 173, # South
+	160, 222, 220, 81, 85, 143, 162, 133, 132, 44, 147, 146, 131, 79, 206, 204, 200, 18, # East
+	174, 148, 163, 164, 165, 55, 120, 42, 138, 130, 197, 199, 202, 19, 198, 45, 203, 207, # West
+	82, 86, 90, 91, 99, 100, 93, 94, 95, 87, 97, 98, 77, 78, 83, 84, 104, 103, 102, 101, # Bottom
+	111, 107, 106, 115, 105, 123, 50, 125, 126, 129, 114, 118, 48, 145, 119, 223, 36, 51, 71, 187, 16, 205, 35, 228, 75, # TopLow
+	121, 40, 39, 128, 127, 178, 144, 180, 141, 179, 109, 116, 47, 49, 117, 185, 184, 183, 72, 216, 213, 212, 211, 210, 209  # TopHigh
+      ]
+    }
+    streams:[slow, fast, archiving]
```

Comment:
Do we keep all three streams?

Reply:
For now, let's do that - this just exposes data from the frontend more frequently. Not opposed to shifting back to `archiving` stream once all testing is done, but shifter can switch between streams on the frontend so it's an innocuous change

Reply:
Okay fab - sounds fine to me!